pytest/test_bem_linear_combination_of_dofs.py

```diff
@@ -5,11 +5,11 @@
 import capytaine as cpt
 import pytest 
 
-solver = cpt.BEMSolver()
 method = ['indirect','direct']
 
 @pytest.mark.parametrize("method", method)
 def test_sum_of_dofs(method):
+    solver = cpt.BEMSolver()
```

Comment:
That's weird. Reinitializing the solver at the beginning of the test should not change its behavior.

Reply:
I think I understand the issue. It is unrelated with this PR, it comes from #420, I'll open another PR for this.

Reply:
See #442